app/utils/dscp-api/index.js

```diff
@@ -4,10 +4,11 @@ const FormData = require('form-data')
 const { DSCP_API_PORT, DSCP_API_HOST } = require('../../env')
 
 module.exports = {
-  async runProcess(payload, authToken) {
+  async runProcess({ file, ...payload }, authToken) {
     const url = `http://${DSCP_API_HOST}:${DSCP_API_PORT}/v3/run-process`
     const formData = new FormData()
     formData.append('request', JSON.stringify(payload))
+    if (file) formData.append('file', file, payload.filename)
```

Comment:
thanks

Reply:
example of building a request with multiple files:
https://github.com/digicatapult/inteli-demo/blob/9b2e12866d8abbc7c07df639ba9c800fd4296f58/src/utils/runProcessFormat.js#L85

Reply:
@mattdean-digicatapult in regards to file this is all good, as first argument is a type, second is expected to be a blob/buffer and a third file name, then dscp-api checks for path and filename

Reply:
Leaving required certs as literal for time being is just mainly because i think it involves a few changes there and there so it makes sense to keep as a separate story.